apps/site/lib/site_web/controllers/fare_transformation_controller.ex

```diff
@@ -0,0 +1,64 @@
+defmodule SiteWeb.FareTransformationController do
+  @moduledoc """
+  Controller for the Fare Transformation section of the website.
+  """
+  use SiteWeb, :controller
+  alias Fares.ProposedLocations
+  alias Stops.{Nearby}
+  import SiteWeb.ViewHelpers, only: [cms_static_page_path: 2]
+
+  def index(conn, %{"id" => "proposed-sales-locations"} = params) do
+    finder(conn, params)
+  end
+
+  def index(conn, _params) do
+    redirect(conn,
+      to: SiteWeb.Router.Helpers.fare_path(conn, :show, "retail-sales-locations")
+    )
+  end
+
+  def finder(conn, %{"location" => _location} = params) do
+    {position, formatted} =
+      SiteWeb.FareController.calculate_position(params, &GoogleMaps.Geocode.geocode/1)
+
+    nearby_proposed_locations =
+      Nearby.nearby_with_varying_radius_by_mode(position)
+      |> Enum.map(fn stop -> stop.id end)
+      |> ProposedLocations.by_stop_ids()
```

Comment:
⚠️  The `Nearby.nearby_with_varying_radius_by_mode(position)` isn't catching all possible PSLs.

Search "Boston Landing", and the PSL page will only return one option on Arthur St @ Guest St, but there is a PSL pin in the ArcGIS data directly at Boston Landing.  When I hardcode the logic in `ProposedLocationsby_stop_id()` to also include the id "Boston Landing", the PSL shows up on the page.  So there's something about the `Nearby` logic that isn't finding this station that is very close by

Reply:
Does the query to ArcGIS need to be based on stops, or can it be based on lat-lon?  

I believe ArcGIS can be queried to "get all PSLs within 10 miles of this location."  If we use this query format and just pass in the user input as the central location, than we can skip this `Nearby` step